content/examples.md

```diff
@@ -232,24 +232,24 @@ from the previous exercise.
 Copy-paste the code into a cell
 
 Install the line profiler
-```python
+```
 !pip install line_profiler
 ```
 Load the IPython extension
 ```python
 %load_ext line_profiler
```

Comment:
So here there is no warnings? I don't fully understand when you get warning and when you don't but I guess it is OK!

Reply:
I don't fully understand either.